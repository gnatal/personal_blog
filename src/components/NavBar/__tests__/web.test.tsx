import React from 'react';
import renderer from 'react-test-renderer';
import NavBarWeb from '../web';

test('It should write an span in the nav header', () => {
  const component = renderer.create(<NavBarWeb />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
