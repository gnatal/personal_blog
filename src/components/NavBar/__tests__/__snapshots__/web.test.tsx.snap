// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should write an span in the nav header 1`] = `
<div
  className="w-full hidden flex-grow sm:flex sm:items-center sm:w-auto sm:block"
>
  <div
    className="text-sm sm:flex-grow"
  >
    <a
      className="block mt-4 md:mt-2 text-md sm:inline-block lg:mt-0 text-teal-100 hover:text-white mr-4"
      href="/"
    >
      in√≠cio
    </a>
    <a
      className="block mt-4 md:mt-2 text-md sm:inline-block lg:mt-0 text-teal-100 hover:text-white mr-4"
      href="/about"
    >
      sobre
    </a>
    <a
      className="block mt-4 md:mt-2 text-md sm:inline-block lg:mt-0 text-teal-100 hover:text-white mr-4"
      href="/posts"
    >
      blog
    </a>
  </div>
  <div>
    <a
      className="inline-block text-sm px-4 py-2 leading-none border rounded text-white border-white hover:border-transparent hover:text-teal-500 hover:bg-white mt-4 md:mt-0"
      href="/login"
    >
      login
    </a>
  </div>
</div>
`;
